README.md

```diff
@@ -19,3 +19,31 @@ Github Pages only runs in safe mode, preventing the usage of [symlinks](https://
 1. Copy the contents of `node_modules/@patternfly/patternfly` to `_includes/patternfly`.
 1. Delete `_includes/patternfly/assets` and `_includes/patternfly/icons` as they are not needed.
 1. Commit the changes.
+
+## Deployment
+
+### OKD
+
+1. Copy the generated source in `_site/okd/errors.html` to the corresponding template at https://github.com/openshift/oauth-server/blob/master/pkg/server/errorpage/templates.go.
+1. Copy the generated source in `_site/okd/login.html` to the corresponding template at https://github.com/openshift/oauth-server/blob/master/pkg/server/login/templates.go.
+1. Copy the generated source in `_site/okd/providers.html` to the corresponding template at https://github.com/openshift/oauth-server/blob/master/pkg/server/selectprovider/templates.go.
+1. Submit a pull request to https://github.com/openshift/oauth-server containing the copied changes.
+
+To test the changes:
+
+1. Deploy an image containing the changes in the pull request.
```

Comment:
I think this needs to be more detailed, but I need help filling in the details.  @stlaz can you help?